src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * @title WET-BOEW Bubble plugin container
+ * @overview Plugin used to open a overlay in the middle of the page
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp @cTopCanada @GormFrank
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "wb-bubble",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	isNotif,
+	$document = wb.doc,
+	i18nDict = {
+		en: {
+			"bubble-skip": "Skip to",
+			"bubble-closeNotif": "Close notification"
+		},
+		fr: {
+			"bubble-skip": "Passer à",
+			"bubble-closeNotif": "Fermer la notification"
+		}
+	},
+
+	/**
+	 * Initiate chat wizard bubble
+	 * @method adjustBubbleMargin
+	 * @param {jQuery DOM element} $selector Element which is the actual bubble
+	 */
+	adjustBubbleMargin = function( $selector ) {
+
+		var $footer = $( "#wb-info" );
+
+		// Hide basic form on load, show chat bubble instead
+		$selector.fadeIn( "slow" );
```

Comment:
See what happens if you do not have this line in combination with not having the `display: none` in the _base.scss file at line 113.

Another option would be to just use ".show()".

Lastly, you could look at how the GC Subway provisional feature prevents blinking on load, by using a new `.no-blink` CSS class ([see GC Subway CSS](https://github.com/wet-boew/GCWeb/blob/master/src/plugins/gc-subway/_screen-md-min.scss#L20) and [GC Subway JS](https://github.com/wet-boew/GCWeb/blob/master/src/plugins/gc-subway/gc-subway.js#L97)) or forcing the bubble to show on basic HTML mode ([see GC subway CSS](https://github.com/wet-boew/GCWeb/blob/master/src/plugins/gc-subway/_screen-md-min.scss#L52)).